index.html

```diff
@@ -59,6 +59,46 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        As the popularity of devices with immersive capabilities continues to
+        expand, there comes an increasing desire to make use of them to consume
+        three-dimensional content across a range of use cases - and a spectrum
+        of complexity. With that in mind, it is necessary to augment the web
+        with appropriate standards to meet those needs at every point along the
+        spectrum. While solutions like [[webxr]] are necessary for more complex
+        experiences in this regard, the level of information disclosure
+        required for such a session should not be necessary for the basic,
+        spatial display of an object in the context of a page. This is the area
+        of focus for the HTML `&lt;model&gt;` element.
```

Comment:
I feel like framing this as only a privacy preserving mechanism is reductive. Ease of use and integration with OS/Browser compositing have also been raised as primary motivations.